frontend/js/schema-conversion.js

```diff
@@ -272,6 +272,99 @@ const globalEditHandler = () => {
   jQuery('#globalDataTypeModal').modal();
```

Comment:
```
createEditDataTypeTable();
```
Here we have called createEditDataTypeTable() function, please check what is the use of calling that function from here

Reply:
we need this now as we removed the function call from html